src/commcare_cloud/commands/terraform/aws.py

```diff
@@ -150,3 +159,116 @@ def run(self, args, unknown_args):
 
         with open(environment.paths.inventory_ini, 'w') as f:
             f.write(out_string)
+
+
+DEFAULT_SIGN_IN_DURATION_MINUTES = 30
+
+
+class AwsSignIn(CommandBase):
+    command = 'aws-sign-in'
+    help = """
+        Use your MFA device to "sign in" to AWS for <duration> minutes (default {})
```

Comment:
What does MFA mean?

Reply:
Multi Factor Auth

Reply:
Curious why you're using MFA and 2FA somewhat interchangeably? Does this mean we actually have the option of more factors than just 2 or could "2FA" be used everywhere "MFA" is in this PR?

Reply:
I'm not particularly biased toward one or the other. I think the most important thing is to be consistent when we talk about it, especially in documentation.